ru/lessons/specifics/ecto.md

```diff
@@ -0,0 +1,342 @@
+---
+layout: page
+title: Ecto
+category: specifics
+order: 2
+lang: ru
+---
+
+Ecto - официальный проект от создателей Elixir. Это оболочка, которая предоставляет возможность коммуникации с базой данных. Ecto позволяет создавать миграции, объявлять модели, вносить и обновлять данные, а также посылать запросы к ним.
+
+## Содержание
+
+- [Установка](#section-1)
+  - [Репозиторий](#section-2)
+  - [Супервизор](#section-3)
+  - [Настройка](#section-4)
+- [Mix задачи](#section-5)
```

Comment:
Just run application locally (`bundle exec jekyll s`, I guess) and check the link correctness.

Ping me if you need any help on that!